packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -32,7 +32,7 @@ import {
   getXoaState,
   isAdmin,
 } from 'selectors'
-import { every, forEach, identity, isEmpty, isEqual, map, pick, size, some } from 'lodash'
+import { countBy, every, forEach, identity, isEmpty, isEqual, map, pick, size, some } from "lodash";
```

Comment:
Was this formatted by Prettier? It should be `...'lodash'`

Reply:
Sorry, maybe a conflict with the IDE, I fix that

Reply:
No problem! :ok_hand: 
But it should have been fixed automatically by commit hooks, even if your IDE has a different configuration, that's why I was wondering.